packages/logo/src/ProductLogos.tsx

```diff
@@ -0,0 +1,463 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+type ProductLogoProps = HTMLElementProps<'svg'> & {
+  knockout?: boolean;
+  size?: number;
+};
+
+export function CloudManagerLogo({
+  size = 18,
+  knockout = false,
+  className,
+  ...rest
+}: ProductLogoProps) {
+  return (
+    <svg
+      {...rest}
+      width={`${size}px`}
```

Comment:
Size here doesn't need to specify `"px"` – it can just be `width={size}`.

Reply:
👍